src/blockmap.jl

```diff
@@ -111,12 +110,11 @@ end
 # vcat
 ############
 """
-    vcat(As::Union{LinearMap,UniformScaling}...)
+    vcat(As::Union{LinearMap,UniformScaling}...)::BlockMap
 
-Construct a `BlockMap <: LinearMap` object, a (lazy) representation of the
-vertical concatenation of the arguments. `UniformScaling` objects are promoted
-to `LinearMap` automatically. To avoid fallback to the generic [`Base.vcat`](@ref),
-there must be a `LinearMap` object among the first 8 arguments.
+Construct a (lazy) representation of the vertical concatenation of the arguments.
+`UniformScaling` objects are promoted to `LinearMap` automatically. To avoid fallback
+to the generic `Base.vcat`, there must be a `LinearMap` object among the first 8 arguments.
```

Comment:
Is this still the case, or do we recommend explicitly wrapping all `AbstractMatrix` arguments with `LinearMap`?

Reply:
Ah this is probably if combining only `LinearMap` with `UniformScaling`.